LoadPanel: Add Show on Page Load How To topic (#7504)
DevExpress/devextreme-documentation: concepts/05 UI Components/LoadPanel/20 Show on Page Load.md

To display a LoadPanel on page load, initialize the component with [visible](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#visible) set to `true`. You can hide the LoadPanel once your page or data finishes loading. The following code snippet demonstrates how to hide the component after [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/Configuration/) finishes loading data:

---

##### jQuery

    <!-- tab: index.js -->
    $(() => {

>         let data = new DevExpress.data.ODataStore({

Reviewer: ```suggestion
        const data = new DevExpress.data.ODataStore({
```

    <template>
        <DxLoadPanel
            :visible="loadingVisible"
            shading-color="rgba(0,0,0,0.4)"
        />
    </template>

    <script setup lang="ts">
    import { DxLoadPanel } from 'devextreme-vue/load-panel';
    import ODataStore from 'devextreme/data/odata/store';

    const loadingVisible = ref(true);

    const data: ODataStore = new ODataStore({
        url: "https://js.devexpress.com/Demos/DevAV/odata/Products"
    });

    data.load().then(() => {
        loadingVisible.value = false;
    });
    </script>

##### React

    <!-- tab: App.tsx -->
    import { LoadPanel } from 'devextreme-react/load-panel';
    import ODataStore from 'devextreme/data/odata/store';

    export default function App () {
        const [loadingVisible, setLoadingVisible] = useState(true);

        const data: ODataStore = new ODataStore({
            url: "https://js.devexpress.com/Demos/DevAV/odata/Products"
        });

        data.load().then(() => {
            setLoadingVisible(false);
        });

        return (
            <LoadPanel
                shadingColor="rgba(0,0,0,0.4)"
                visible={loadingVisible}
            />
        )
    }

---

> [note] The default behaviour of LoadPanel is to display over the entire application [window](https://developer.mozilla.org/en-US/docs/Web/API/Window). To place the component over a specific area, define the [position](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#position) property.

Reviewer 2: ```suggestion
[note] LoadPanel is displayed over the entire application [window](https://developer.mozilla.org/en-US/docs/Web/API/Window) by default. To place the component over a specific area, define the [position](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#position) property.
```

Reviewer 2: ```suggestion
[note] LoadPanel is displayed over the entire application [window](https://developer.mozilla.org/en-US/docs/Web/API/Window). To place the component over a specific area, define the [position](/Documentation/ApiReference/UI_Components/dxLoadPanel/Configuration/#position) property.
```
